esp/services/ws_decoupledlogging/ws_decoupledloggingservice.hpp

```diff
@@ -23,6 +23,29 @@
 #include "environment.hpp"
 #include "logthread.hpp"
 
+struct LogAgentAction
+{
+    CLogAgentActions type;
+    StringArray* fileNames;
```

Comment:
This looks a bit strange, and it should be initialised to nullptr.  It would benefit from a constructor for cleaning up the calling code.
(Or you could use LogAgentAction action{CLogAgentActions_CleanAckedLogFiles};)

Reply:
I will initialise it to nullptr.